docs/source/docs/quick-start/quick-install.md

```diff
@@ -0,0 +1,36 @@
+# Quick Install
+
+## Install the latest image of photonvision for your coprocessor
+
+- For the supported coprocessors
+  - RPI 3,4,5
+  - Orange Pi 5
+  - Limelight
+
+[Download](https://github.com/PhotonVision/photonvision/releases/latest) the latest preconfigured image of photonvision for your coprocessor
+
+| Coprocessor          | Image filename                                       | Jar                                   |
+| -------------------- | ---------------------------------------------------- | ------------------------------------- |
+| OrangePi 5           | photonvision-{version}-linuxarm64_orangepi5.img.xz   | photonvision-{version}-linuxarm64.jar |
```

Comment:
Is there a magic GitHub link that will always take me to the latest version of this artifact?

Reply:
there should be a link that would just automatically download it if that is what we want